Formula/crosstool-ng.rb

```diff
@@ -46,6 +47,7 @@ def install
     ENV["BISON"] = "#{Formula["bison"].opt_bin}/bison"
     ENV["M4"] = "#{Formula["m4"].opt_bin}/m4"
     ENV["MAKE"] = "#{Formula["make"].opt_bin}/gmake"
+    ENV["PYTHON"] = "python"
```

Comment:
```suggestion
    ENV["PYTHON"] = "#{Formula["python@3.9"].opt_bin}/python3"
```
Without this, I think you'll pick up the system python2 interpreter.

Reply:
Ah. Thanks!